README.md

```diff
@@ -1 +1,131 @@
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project">
+Hello , this a mobile first portfolio project</a>
+
+[Basir elham's portfolio] This a portfolio mobile first app. It is created according to a figma template and the complete version will be pushed next time.
+
+## 🛠 Built With 
+
+### Tech Stack 
+HTML
+CSS
+
+### Key Features 
+
+- **[Navbar]**
+- **[Background image]**
+- **[Bobile first]**
+- **[Linter is used]**
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:You have to clone this project to your local machine
+
+### Setup
+
+Clone this repository to your desired folder:git clone git@github.com:Elhambasir/MyPortfolio.git
+
+### Install
+
+Install this project with: Any software which you are comfortable with
+
+### Usage
+
+To run the project, execute the following command: On your computer's keyboard press f5 or run it with live server
+
+### Run tests
+
+To run tests, run the following command: In visual studio code with live server plugin right click and run live server.
+
+### Deployment
+
+You can deploy this project using: GitHub
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author**
+
+- GitHub: [@Elhambasir](https://github.com/Elhambasir)
+- LinkedIn: [basir ahmadi](https://www.linkedin.com/in/basir-ahmadi-8aba89263)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- FUTURE FEATURES -->
+## Future Features
+
+- [ ] **[Responsive]**
+- [ ] **[Other features]**
+- [ ] **[Better style]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+## Contributions
+
+Contributions, issues, and feature requests are welcome!
+
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+> Write a message to encourage readers to support your project
+
```

Comment:
Kindly, remove this `line` in your `README.md file`
so as your it won't look like a copy paste :wink: